openedx/core/djangoapps/content_tagging/tests/test_tasks.py

```diff
@@ -5,27 +5,53 @@
 
 from unittest.mock import patch
 
-from django.core.management import call_command
 from django.test import override_settings
 from edx_toggles.toggles.testutils import override_waffle_flag
-from openedx_tagging.core.tagging.models import LanguageTaxonomy, Tag, Taxonomy
+from openedx_tagging.core.tagging.models import Tag, Taxonomy
 from organizations.models import Organization
 
 from common.djangoapps.student.tests.factories import UserFactory
 from openedx.core.djangolib.testing.utils import skip_unless_cms
 from xmodule.modulestore.tests.django_utils import TEST_DATA_SPLIT_MODULESTORE, ModuleStoreTestCase
 
 from .. import api
-from ..models import TaxonomyOrg
+from ..models.base import TaxonomyOrg
 from ..toggles import CONTENT_TAGGING_AUTO
 from ..types import ContentKey
 
 LANGUAGE_TAXONOMY_ID = -1
 
 
+class LanguageTaxonomyTestMixin:
+    """
+    Mixin for test cases that expect the Language System Taxonomy to exist.
+    """
+
+    def setUp(self):
+        """
+        When pytest runs, it creates the database by inspecting models, not by
+        running migrations. So data created by our migrations is not present.
+        In particular, the Language Taxonomy is not present. So this mixin will
+        create the taxonomy, simulating the effect of the following migrations:
+            1. openedx_tagging.core.tagging.migrations.0012_language_taxonomy
+            2. content_tagging.migrations.0007_system_defined_org_2
+        """
+        super().setUp()
+        Taxonomy.objects.get_or_create(id=-1, defaults={
+            "name": "Languages",
+            "description": "Languages that are enabled on this system.",
+            "enabled": True,
+            "allow_multiple": False,
+            "allow_free_text": False,
+            "visible_to_authors": True,
+            "_taxonomy_class": "openedx_tagging.core.tagging.models.system_defined.LanguageTaxonomy",
+        })
+        TaxonomyOrg.objects.create(taxonomy_id=-1, org=None)
```

Comment:
Could this have a `get_or_create` too, so we're not tripped up in future by multiple entries created during testing?

```suggestion
        TaxonomyOrg.objects.get_or_create(taxonomy_id=-1, defaults={"org": None})
```